src/vs/workbench/contrib/chat/browser/chatParticipantContributions.ts

```diff
@@ -69,7 +69,7 @@ const chatParticipantExtensionPoint = extensionsRegistry.ExtensionsRegistry.regi
 					type: 'string'
 				},
 				disambiguation: {
-					description: localize('chatParticipantDisambiguation', "Metadata to help with automatically routing user questions to this chat participant."),
+					description: localize('chatParticipantDisambiguation', "Metadata to help with automatically routing user questions to this chat participant. You must add `contribChatParticipantDetection` to `enabledApiProposals` to use this API."),
```

Comment:
fyi you can use `markdownDescription` so the codespans will be rendered as markdown